test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/ProcessResult.cs

```diff
@@ -3,9 +3,9 @@
 
 namespace Microsoft.AspNetCore.Razor.Design.IntegrationTests
 {
-    internal class MSBuildResult
+    internal class ProcessResult
```

Comment:
I don't think we should change this. This isn't supposed to be general purpose, it's use in a set of assert methods that target MSBuild explicitly.